docs/refs/api/drivers/BusInOut.md

```diff
@@ -1,9 +1,13 @@
 #### BusInOut
 
-Use the BusInOut interface as a bidirectional bus that collects a number of [DigitalInOut](DigitalInOut.md) pins that can be read and written as one value.
+BusInOut is an abstraction that takes up to 16 [DigitalInOut](DigitalInOut.md) pins and makes them appear as though they are linearly memory mapped for ease of use. This abstraction is useful for reading or writing multiple pins at the same time. In general this abstraction can be used to make code less cluttered, clearer, and take less time to write. 
 
 You can use any of the numbered Arm Mbed pins as a [DigitalInOut](DigitalInOut.md).
 
+**Tips:**
+* You can have up to 16 pins in a Bus. 
+* The order of pins in the constructor is the reverse order of the pins in the byte order. So if you have BusInOut(a,b,c,d) then the order of bits in the nibble would be `dcba` with `a` being bit 0, `b` being bit 1, `c` being bit 2 and so on.
```

Comment:
- Please change "So if you have..." to "If you have..."
- Please put "BusInOut(a,b,c,d)" in code format.
- Please explain "nibble". This is the first time we use it.
- In the last sentence, please change "would be" to "is" for consistent tense.

Reply:
Byte is 8bits, nibble is 4 bits, byte is 1 bit. Common knowledge among engineers. 

Are you able to directly modify this?